connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedKafkaCluster.java

```diff
@@ -337,18 +337,28 @@ public void createTopic(String topic) {
      * @param topic The name of the topic.
      */
     public void createTopic(String topic, int partitions) {
-        createTopic(topic, partitions, 1, new HashMap<>());
+        createTopic(topic, partitions, 1, Collections.emptyMap(), new Properties());
```

Comment:
It seems we don't need to pass `new Properties()`, right?

Reply:
yes, you're right! Sorry, I didn't catch your idea. Updated. Thanks.